vm/devices/storage/disk_nvme/nvme_driver/src/queue_pair.rs

```diff
@@ -668,15 +668,15 @@ impl Issuer {
             .await
             .expect("pool cap is sufficient");
 
-        assert_eq!(
-            mem.page_count(),
-            1,
-            "larger requests not currently supported"
+        assert!(
+            mem.page_count() <= 4096,
+            "Requests larger than 4096 pages not currently supported"
```

Comment:
I'm not sure that we need this assert here. If `make_prp` can't handle it, then we should put the assert there.